internal/controller/drplacementcontrolvolsync.go

```diff
@@ -150,6 +150,8 @@ func (d *DRPCInstance) createOrUpdateVolSyncDestManifestWork(srcCluster string)
 			return ctrlutil.OperationResultNone, err
 		}
 
+		// TODO: Update peerClass from source VRG MW (NOT from view)
```

Comment:
@BenamarMk I still need to address this for Secondary (a separate commit) and then we can look at merging it etc. (just a heads up!)

Reply:
@BenamarMk added [commit](https://github.com/RamenDR/ramen/pull/1638/commits/6cf2df4a4d3abef6cc6b7a73479eb6b823988dac) to address the above.